doc/source/examples/analysis/reduced_dimensions/pca.ipynb

```diff
@@ -37,7 +37,7 @@
     {
```

Comment:
<p>All these <code>last executed</code> bits need to be updated or removed. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/UserGuide/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>